support-frontend/assets/components/checkoutNudge/checkoutNudge.tsx

```diff
@@ -45,52 +46,50 @@ const top = css`
 const topheading = css`
 	margin-top: ${space[3]}px;
 `;
-const heading = (backColor: string, marginBottom: number) => css`
-	margin-left: 10px;
-	color: ${backColor};
-	${headline.xxsmall({ fontWeight: 'bold', lineHeight: 'tight' })};
-	line-height: 108%;
+const heading = (
+	backColor: string,
```

Comment:
Had assumed this would be for a background colour so maybe something like this instead?

```suggestion
	color: string,
```

Reply:
Ok have passed through color